src/OpenTelemetry/README.md

```diff
@@ -57,264 +50,27 @@ which provides the ability to enrich logs emitted with `ILogger` with
-  : This is a processor which can be composed from multiple processors. This is
-  typically used to construct multiple processing pipelines, each ending with
-  its own exporter.
-
-* [SimpleExportProcessor&lt;T&gt;](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#simple-processor)
-  : This is an exporting processor which passes telemetry to the configured
-  exporter without any batching.
-
-Follow [this](../../docs/trace/extending-the-sdk/README.md#processor) document
-to learn about how to write own processors.
-
-*The processors shipped from this SDK are generics, and supports tracing and
-logging, by supporting `Activity` and `LogRecord` respectively.*
-
-### Resource
-
-[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
-is the immutable representation of the entity producing the telemetry. If no
-`Resource` is explicitly configured, the default is to use a resource indicating
-this [Telemetry
-SDK](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk).
-The `ConfigureResource` method on `TracerProviderBuilder` can be used to set a
-configure the resource on the provider. When the provider is built, it automatically
-builds the final `Resource` from the configured `ResourceBuilder`. As with
-samplers, there can only be a single `Resource` associated with a provider.
-It is not possible to change the resource builder *after* the provider is
-built, by calling the `Build()` method on the `TracerProviderBuilder`.
-`ResourceBuilder` offers various methods to construct resource comprising
-of multiple attributes from various sources.
-
-The snippet below shows configuring a custom `ResourceBuilder` to the provider.
-
-```csharp
-using OpenTelemetry;
-using OpenTelemetry.Resources;
-using OpenTelemetry.Trace;
-
-using var tracerProvider = Sdk.CreateTracerProviderBuilder()
-    .ConfigureResource(r => r.AddService("MyServiceName"))
-    .Build();
-```
-
-It is also possible to configure the `Resource` by using following
-environmental variables:
-
-| Environment variable       | Description                                        |
-| -------------------------- | -------------------------------------------------- |
-| `OTEL_RESOURCE_ATTRIBUTES` | Key-value pairs to be used as resource attributes. See the [Resource SDK specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.5.0/specification/resource/sdk.md#specifying-resource-information-via-an-environment-variable) for more details. |
-| `OTEL_SERVICE_NAME`        | Sets the value of the `service.name` resource attribute. If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence. |
-
-### Sampler
+minutes](../../docs/metrics/getting-started/README.md) to get up and running.
```

Comment:
```suggestion
minutes](../../docs/metrics/getting-started/README.md) to get metrics up and running.
```

Reply:
It says "metrics" at the begging of the sentence so it felt a bit redundant to say it at the end as well. I tweaked the whole thing a bit. LMK how you feel about it now.

Reply:
Looks good!